docs/_static/css/light.css

```
@@ -0,0 +1,31 @@
+/* links */
+.rst-content a{
+    color: #002fa7;
+}
+
+.rst-content a:visited {
+    color: #002fa7;
+}
+
+.rst-content .highlighted {
+	background: #eac545;
+}
+
+/* code blocks highlights */
+.rst-content pre {
+	background: #e5e5e5;
+}
+
+.wy-side-nav-search img{
+	width: 80px !important;
+}
```

I think this should be more specific. We only want this height specifically for the logo:
```suggestion
.wy-side-nav-search img.logo {
	width: 80px !important;
}
```

---

Also, why is this in `light.css`? Don't we need this rule in dark mode as well?